pep-0394.txt

```diff
@@ -43,11 +44,14 @@ Recommendation
 * When  invoked, ``python2`` should run some version of the Python 2
   interpreter, and ``python3`` should run some version of the Python 3
   interpreter.
-* The more general ``python`` command should be installed whenever
-  any version of Python 2 is installed and should invoke the same version of
+* If the ``python`` command is installed, it should invoke the same version of
   Python as the ``python2`` command (however, note that some distributions
   have already chosen to have ``python`` implement the ``python3``
-  command; see the `Rationale`_ and `Migration Notes`_ below).
+  command, and system administrators may be allowed to change the default;
```

Comment:
I don't want sysadmins to feel empowered by this PEP, unless they are the only user of a system. Sysadmins often don't know what their users are doing. Changing what `python` does should be up to the user. The sysadmin should not have any dependencies on `python` -- sysadmin scripts should use `python2` or `python3`.